echo/readme.md

```
@@ -1,175 +1,175 @@
+`Can't open the append-only file: Permission denied`
+
+If your Redis container fails to start and you see a “Permission denied” error about the append-only file, you may need to change permissions on the Redis data folder.
+
+1. **Open a local WSL terminal** (outside of the container).
+2. **Run**:
+   ```bash
+   sudo chown -R 1001:1001 ./echo/.devcontainer/redis_data
+   ```
+3. **Restart** the redis container from Docker Desktop.
+
+### Minio not starting 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)